billboard_bronze.cpp

```diff
@@ -0,0 +1,77 @@
+//Jeffrey Chen
+//10 minutes, ****x *****
+//Difficulty - 4/10
+//It didn't seem hard, though I'm not sure why the fifth test case isn't working.

```

Comment:
You have 50 min left so you should tried to debug it and feedback on your debugging. Debugging is the larger part of programming.